aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -40,7 +37,7 @@ class NodejsNpmEsbuildWorkflow(BaseWorkflow):
 
     CAPABILITY = Capability(language="nodejs", dependency_manager="npm-esbuild", application_framework=None)
 
-    EXCLUDED_FILES = (".aws-sam", ".git")
+    EXCLUDED_FILES = (".aws-sam", ".git", "node_modules")
```

Comment:
we used to concatenate node_modules when excluding, so I thought it might make more sense to just add it here if we always exclude it.

Reply:
I didn't find any in the esbuild workflow. The excludes was used in one place, and it appended the node modules every time.